library/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 
-          package="com.mindera.skeletoid"
-        >
+    package="com.mindera.skeletoid">
 
+    <provider
```

Comment:
I think we should for several reasons:

1. Even if it isn't used, it doesn't have any downsides
2. It is less error prone
3. It works out of the box.

I don't see any disadvantages, do you?